internal/trigger/messagebus/messaging.go

```diff
@@ -122,6 +122,15 @@ func (trigger *Trigger) Initialize(appWg *sync.WaitGroup, appCtx context.Context
 						logger.Trace("Published message to bus", "topic", trigger.Configuration.Binding.PublishTopic, clients.CorrelationHeader, msgs.CorrelationID)
 					}
 				}()
+			case bg := <-background:
+				go func() {
+					err := trigger.client.Publish(bg, trigger.Configuration.Binding.PublishTopic)
+					if err != nil {
+						logger.Error(fmt.Sprintf("Failed to publish background Message to bus, %v", err))
```

Comment:
Add return so Trace message isn't logged when this error occurs. Yes, we need a return in the above case also. ;-)

Reply:
I added the missing one too :)